vllm/core/scheduler.py

```diff
@@ -1200,6 +1214,21 @@ def _schedule_chunked_prefill(self) -> SchedulerOutputs:
 
         # Update swapped requests.
         self.swapped.extend(running_scheduled.swapped_out)
+
+        blocks_to_copy = running_scheduled.blocks_to_copy
+        blocks_to_copy.extend(swapped_in.blocks_to_copy)
+
+        blocks_to_swap_in = swapped_in.blocks_to_swap_in
+        blocks_to_swap_out = running_scheduled.blocks_to_swap_out
+
+        # NOTE(Kuntai): extend the swapping list for CPU offloading
+        new_swap_out, new_swap_in = \
+                self.block_manager.get_and_reset_swaps(time.time())
```

Comment:
However, the `get_and_reset_swaps` is called directly here without `get_physical_block_id`. I think these block IDs are sent to the cache engine directly later, so not the real physical block IDs.

Reply:
Got it! I double-checked the logic and you are right. Just pushed another commit to fix the issue and update the docstring. Thanks for the catch!